sqlmesh/core/state_sync/engine_adapter.py

```diff
@@ -103,17 +104,20 @@ def __init__(
         self.plan_dags_table = exp.table_("_plan_dags", db=self.schema)
         self.versions_table = exp.table_("_versions", db=self.schema)
 
+        index_type = index_text_type(self.engine_adapter.dialect)
+        blob_type = blob_text_type(self.engine_adapter.dialect)
+
         self._snapshot_columns_to_types = {
-            "name": exp.DataType.build("text"),
-            "identifier": exp.DataType.build("text"),
-            "version": exp.DataType.build("text"),
-            "snapshot": exp.DataType.build("text"),
-            "kind_name": exp.DataType.build("text"),
+            "name": exp.DataType.build(index_type),
```

Comment:
@GeorgeSittas any reason we didn't update this before?

Reply:
update what?

Reply:
Hmm I'm not fully sure - maybe an oversight?